### DISCUSSION TITLE
have an problem with double quotes font when rendering

### DISCUSSION THREAD

**Discussion by fristzzz** [quarto-dev/quarto-cli#9644] [Q&A] [answered]
### Description

I'm writing Chinese pdf document  and I want Chinese and its signs to be FangSong font, while Times New Roman for English and signs. But I found that double quotes are Times New Roman after rendering. 

I enabled the *keep-tex* and found that the quotes in .tex file had become `` ''.  I wonder if there is any way to solve this problem.

Here is main part of my font matter

```qmd
---
title: xxx
subtitle: xxx
author:  xxx
fig-align: center
fig-cap-location: bottom
tbl-cap-location: top
reference-section-title: 参考文献
highlight-style: github
bibliography: citation.bib
link-citations: true
csl: apa.csl
toc: true
toc-depth: 2
toc-title: 目录
crossref: 
  chapters: true
  fig-title: 图
  fig-prefix: 图
  tbl-title: 表
  tbl-prefix: 表
  eq-title: 等式
  eq-prefix: 等式
abstract: xxx
keywords: xx
format:
  pdf:
    documentclass: article
    number-sections: true
    number-depth: 3
    fig-align: center
    mainfont: Times New Roman
    CJKmainfont: FangSong
    keep-tex: true
    include-in-header: 
      text: |        
        \usepackage{ctex}
        \usepackage{amsthm,mathrsfs}
        \usepackage{xeCJK}
        \usepackage{geometry}
        \usepackage{fontspec}
        \geometry{a4paper}
        \geometry{margin=1in}
        \usepackage{indentfirst}
        \setlength{\parindent}{2em}
        \renewcommand{\baselinestretch}{1.0}
        \usepackage[fontsize=12pt]{fontsize}
---
```

**Comment by mcanouil** [comment]
That's a LaTeX feature (called ligature) unrelated to Quarto.
You should look on LaTeX specialised platform on how to change this LaTeX behaviour.

**Reply by fristzzz** [reply]
Really grateful for suggesting.
But after I used “ ” to replace `` '' , then used `xelatex file.tex` , it  just worked fine. So I think there may be a way that quarto doesn't transform the fullwith “” so that it can be directly rendered by xelatex and be the font of FangSong. Or some other solution could work. 
I'll try asking for help on some LaTeX platforms. Thanks again for helping.

**Reply by mcanouil** [reply, marked as answer]
You could disable the use of ligature by Pandoc:

````qmd
---
title: "Quarto Playground"
format: latex-smart
---

"This is a playground for Quarto."
````

But I am not sure it would work wiht `format: pdf-smart`.
